lib/releaselib.sh

```diff
@@ -709,7 +709,7 @@ release::gcs::publish () {
 
     # If public, validate public link
     logecho -n "* Validating uploaded version file at $public_link: "
-    contents="$(curl -Ls $public_link)"
+    contents="$(curl --retry=5 -Ls $public_link)"
```

Comment:
I would have done that if there were more curl usages, but this is one of the only bare curls-- the rest mostly use GHCURL that does authenticated retrying github operations.